parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -152,6 +153,8 @@ public static enum JobSummaryLevel {
   public static final String BLOOM_FILTER_EXPECTED_NDV = "parquet.bloom.filter.expected.ndv";
   public static final String BLOOM_FILTER_MAX_BYTES = "parquet.bloom.filter.max.bytes";
   public static final String BLOOM_FILTER_FPP = "parquet.bloom.filter.fpp";
+  public static final String DYNAMIC_BLOOM_FILTER_ENABLED = "parquet.bloom.filter.dynamic.enabled";
```

Comment:
@wgtmac In the end, at least one BloomFilter will be retained, because I won't remove the `largestCandidate` (please see `AdaptiveBlockSplitBloomFilter#insertHash`)